### PR TITLE
Milestone 2: Update tables & add aggregate queries

### PR DESCRIPTION
**Pull request by hmunish**
- Add new 7 animals data
- Add species column for animals table
- Add aggregate search queries for animals

snap-1
snap-2
snap-3
snap-4
snap-5

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,12 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8.00);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', '2020-02-08', 0, false, 11);
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Kindly change the `weight_kg` data to match with the provided, it's a negative value

File: data.sql [hunk 2]
```diff
@@ -4,3 +4,12 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
...
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', '2021-11-15', 2, true, 5.7);
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Kindly change the `weight_kg` data to match with the provided, it's a negative value

File: data.sql [hunk 3]
```diff
@@ -4,3 +4,12 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
...
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', '1993-04-02', 3, false, 12.13);
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Kindly change the `weight_kg` data to match with the provided, it's a negative value

File: data.sql [hunk 4]
```diff
@@ -4,3 +4,12 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
...
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', '2005-06-12', 1, true, 45);
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Kindly change the `weight_kg` data to match with the provided, it's a negative value

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @hmunish  

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

hand-sketching-stair-slogan-one-step-closer-chalkboard-79948193

## Highlights

✅ Linters are passing
✅ Good work implementing requested queries 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (hmunish) [reply data.sql]: @BregornOriginal  I have just amended it.

File: data.sql (hunk 4 above)
Comment (hmunish) [reply data.sql]: @BregornOriginal , I have amended it, please advise.

**Review by danmainah — APPROVED** [review]
Hi @hmunish , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Implemented the change requested to ensure use of correct database figures in this project.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**